frontend/src/scenes/surveys/SurveyOverview.tsx

```diff
@@ -54,13 +52,28 @@ const QuestionIconMap = {
 export function SurveyOverview(): JSX.Element {
     const { survey, selectedPageIndex, targetingFlagFilters } = useValues(surveyLogic)
     const { setSelectedPageIndex } = useActions(surveyLogic)
-    const { featureFlags } = useValues(featureFlagLogic)
+
+    const isExternalSurvey = survey.type === SurveyType.ExternalSurvey
 
     const { surveyUsesLimit, surveyUsesAdaptiveLimit } = useValues(surveyLogic)
     return (
         <div className="flex gap-4">
             <dl className="flex flex-col gap-4 flex-1 overflow-hidden">
-                <SurveyOption label="Display mode">{SURVEY_TYPE_LABEL_MAP[survey.type]}</SurveyOption>
+                <SurveyOption label="Display mode">
+                    <div className="flex flex-col">
+                        <div className="flex flex-row items-center gap-2">
+                            {SURVEY_TYPE_LABEL_MAP[survey.type]}
+                            {isExternalSurvey && <CopySurveyLink surveyId={survey.id} className="w-fit" />}
+                        </div>
+                        {isExternalSurvey && (
+                            <span>
+                                Track responses to users by adding{' '}
```

Comment:
same as https://github.com/PostHog/posthog/pull/36080/files#r2250709670
we are using multiple wording for track/collect/identify response/identify responses to users, etc, lets agree on something and use it everywhere